client/src/components/TheAnalyticsDemoForm.vue

```diff
@@ -0,0 +1,110 @@
+            :items="selectValues.action"
+            item-text="text"
+            item-value="value"
+            label="Action"
+        />
+
+        <v-btn type="submit">Update</v-btn>
+    </form>
+</template>
+
+<script>
+import dayjs from 'dayjs';
+
+export default {
+    props: {
+        maxUsers: {
+            type: Number,
+            required: false,
+            default: 100
+        },
+        maxDaysInFuture: {
+            type: Number,
+            required: false,
+            default: 30
+        }
+    },
+
+    data() {
+        return {
+            form: {
+                date: dayjs().toISOString(),
+                cameFrom: 'facebook',
+                user: 'user1',
+                action: 'homepage'
+            },
+            selectValues: {
+                date: (() => {
+                    const dates = [];
+
+                    for (let i = 0; i < this.maxDaysInFuture; i++) {
+                        const date = dayjs().add(i, 'day');
+
+                        dates.push({
+                            text: date.toString(),
+                            value: date.toISOString()
+                        });
+                    }
+
+                    return dates;
+                })(),
+                cameFrom: [
+                    { text: 'Google Ads', value: 'google' },
+                    { text: 'Facebook Ads', value: 'facebook' },
+                    { text: 'Email', value: 'email' },
+                    { text: 'Direct', value: 'direct' },
+                    { text: 'Referral', value: 'referral' },
+                    { text: 'None', value: 'none' }
+                ],
+                user: (() => {
+                    const users = [];
+
+                    for (let i = 1; i <= this.maxUsers; i++) {
+                        users.push({
+                            text: `User${i}`,
+                            value: `user${i}`
+                        });
+                    }
+
+                    return users;
+                })(),
+                action: [
+                    { text: 'Register', value: 'register' },
+                    { text: 'Visit Homepage', value: 'homepage' },
+                    { text: 'Visit Product1 Page', value: 'product1page' },
+                    { text: 'Visit Product2 Page', value: 'product2page' },
+                    { text: 'Visit Product3 Page', value: 'product3page' },
+                    { text: 'Add Product1 to Cart', value: 'product1cart' },
+                    { text: 'Add Product2 to Cart', value: 'product2cart' },
+                    { text: 'Add Product3 to Cart', value: 'product3cart' },
+                    { text: 'Buy Product1', value: 'product1buy' },
+                    { text: 'Buy Product2', value: 'product2buy' },
+                    { text: 'Buy Product3', value: 'product3buy' }
+                ]
+            }
+        };
+    },
+
+    methods: {
+        submitForm() {
+            console.log(this.form);
```

Comment:
do not merge console.log
you can add toast that with warning that it isnt implemented yet, or just leave `todo`

Reply:
@daniel-pupczyk-rmtcfm-com Sure, I just used console log to check my form working. Gonna delete it.